src/marketplace/find-and-install/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Find and install blocklets'
-description: 'Find and install blocklets'
+description: 'Find and install Blocklets'
```

Comment:
the screenshots in this PR needs to be recreated since the marketplace UI has been polished.

Reply:
before recreating the screenshots, please upgrade your node to latest version: 1.3.8

Reply:
And please help to add a doc on how to upgrade the node to latest verson, there are 2 ways, just install the latest `abtnode/cli` or upgrade form the node settings page.

Reply:
Should I cover this in details about the settings page ?

Reply:
Yes, please cover this in the settings page